app/views/applications/show.html.erb

```diff
@@ -0,0 +1,53 @@
+<section class='app-info'>
+  <h2>Applicant Info</h2>
+  <h3>Name: <%=@app.name%></h3>
+  <p>Address: <%=@app.street_address%></p>
+  <p>City: <%=@app.city%></p>
+  <p>State: <%=@app.state%></p>
+  <p>Zip: <%=@app.zip%></p>
+  <p>Description: <%=@app.description%></p>
+  <p>Status: <%=@app.status%></p>
+</section>
+
+<section class="pets-on-app">
+  <h2>Pets Applied for</h2>
+  <% @app.pets.each do |pet| %>
+    <div id="pet-<%= pet.id %>">
+      <%= link_to "#{pet.name}", "/pets/#{pet.id}", method: :get %>
+    </div>
+  <% end %>
+</section>
+
+<section class='find-pets'>
+  <% if @app.status != "Pending" %>
+    <h2> Add a Pet to this Application </h2>
+    <%= form_with url: "/applications/#{@app.id}", method: :get do |f| %>
+      <%= f.label :search_name, "Search for Pet:" %>
+      <%= f.text_field :search_name %>
+    
+      <%= f.submit "submit"%>
+    <% end %>
+  <% end %>
+</section>
+
+<section class='found-pets'>
+  <% if @found_pets != nil %>
+    <h2> Found pets:</h2>
+    <% @found_pets.each do |pet| %>
+      <div id="pet-<%= pet.id %>">
+        <h4><%= pet.name %></h4>
+        <%=button_to "Adopt this Pet", "/pet_apps?pet_id=#{pet.id}&app_id=#{@app.id}", method: :post%>
+      </div>
+    <% end %>
+  <% end %>
+</section>
+
+<section class='submit-application'>
+  <% if @app.pets.count > 0 %>
+    <%= form_with url: "/applications/#{@app.id}/update", method: :patch do |f| %>
+      <%= f.label :reason_to_adopt, "Why I would make a good owner for these pet(s)?:" %>
+      <p><%= f.text_area :reason_to_adopt %></p>
+      <%= f.submit "Submit Application" %>
+    <% end %>
+  <% end %>
+</section>
```

Comment:
As I was using the app in the browser I noticed 2 bugs. Your application allows for the same pet to be added to the same application more than once, and secondly, once the app is submitted the "Submit Application" button is stll there.